src/test/java/edu/uml/diet/BasicFoodTest.java

```diff
@@ -0,0 +1,145 @@
+package edu.uml.diet;
+
+/**
+ * Created by Kurt Johnson on 2/8/2015.
+ */
+
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+
+public class BasicFoodTest {
+    String foodName;
+    int calories;
+    int fat;
+    int carbs;
+    int protein;
+
+    @Before
+    public void setup() {
+        foodName = "Twinkie";
+        calories = 270;
+        fat = 9;
+        protein = 2;
+        carbs = 46;
+    }
+
+    /**
+     * test to verify that correct  name is returned by getName()
+     */
+    @Test
+    public void testGetName(){
+        BasicFood basicfood = new BasicFood(foodName, calories, fat, carbs, protein);
+        assertEquals("testGetName: positive", foodName, basicfood.getName());
+    }
+
+    /**
+     * negative test for getName() method
+     */
+    @Test
+    public void testGetNameNegative(){
+        BasicFood basicfood = new BasicFood(foodName, calories, fat, carbs, protein);
+        assertFalse("testGetName: negative", "HoHo".equals(basicfood.getName()));
+    }
+
+    /**
+     * test to verify that correct  calories  is returned by getCalories()
+     */
+    @Test
+    public void testGetCalories(){
+        BasicFood basicfood = new BasicFood(foodName, calories, fat, carbs, protein);
+        assertEquals("testGetCalories: positive", calories, basicfood.getCalories());
+    }
+
+    /**
+     * Negative test for getCalories() method
+     */
+    @Test
+    public void testGetCaloriesNegative(){
+        BasicFood basicfood = new BasicFood(foodName, calories, fat, carbs, protein);
+        assertFalse("testGetCalories: negative", (calories+10) == basicfood.getCalories());
+    }
+
+    /**
+     * test to verify that correct  fat  is returned by getFat()
+     */
+    @Test
+    public void testGetFat(){
```

Comment:
Don't forget D.R.Y. - you make basicFood a member variable and initialize it once in a setup method.